src/acquisition/flusurv/flusurv_update.py

```diff
@@ -113,38 +112,185 @@ def update(issue, location_name, test_mode=False):
     # SQL for insert/update
     sql = """
     INSERT INTO `flusurv` (
-      `release_date`, `issue`, `epiweek`, `location`, `lag`, `rate_age_0`,
-      `rate_age_1`, `rate_age_2`, `rate_age_3`, `rate_age_4`, `rate_overall`,
-      `rate_age_5`, `rate_age_6`, `rate_age_7`
+      `release_date`,
+      `issue`,
+      `epiweek`,
+      `location`,
+      `lag`,
+      `season`,
+
+      `rate_overall`,
+
+      `rate_age_0`,
+      `rate_age_1`,
+      `rate_age_2`,
+      `rate_age_3`,
+      `rate_age_4`,
+      `rate_age_5`,
+      `rate_age_6`,
+      `rate_age_7`,
+
+      `rate_age_18t29`,
+      `rate_age_30t39`,
+      `rate_age_40t49`,
+      `rate_age_5t11`,
+      `rate_age_12t17`,
+      `rate_age_lt18`,
+      `rate_age_gte18`,
+
+      `rate_race_white`,
+      `rate_race_black`,
+      `rate_race_hisp`,
+      `rate_race_asian`,
+      `rate_race_natamer`,
+
+      `rate_sex_male`,
+      `rate_sex_female`
     )
     VALUES (
-      %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s
+      %(release_date)s,
```

Comment:
named instead of ordered parameters: 👍👍👍

Reply:
The previous version made me itchy.

Even nicer would be if we could insert each value-name as a pair ("%(release_date)s as `release_date`, %(issue)s as `issue`, ..."). I don't suppose there's a way to do that in SQL. I looked briefly but couldn't find anything.